src/main/java/storage/Storage.java

```diff
@@ -0,0 +1,156 @@
+package storage;
```

Comment:
Maybe add a line after the package statement
```suggestion
package storage;

```